### PR TITLE
fix: remove excess question marks in messages to avoid invalid question parsing

### LINKED ISSUE #2: Fix parsing questions if there are more than 1 question mark
Currently blocked by #1, my idea is to check if there is any question mark, if yes then remove it and do rest of the work

### PR DESCRIPTION
**Pull request by fhodun**
Added question mark removal in message parsing

### CODE REVIEW

File: qp/qp.go [hunk 1]
```diff
@@ -33,6 +34,11 @@ func (qp QuestionParser) ParseString(str string) *Sentence {
 		return nil
 	}
 
+	str, err := utils.RemoveUnwantedCharacters(str)
+	if err != nil {
+		logrus.Errorf("Fail removing polish shit %s\n", err.Error())
```
Comment (gbaranski) [inline qp/qp.go:39]: ~~what about return?~~

File: utils/utils.go [hunk 2]
```diff
@@ -40,6 +42,8 @@ func RemovePolishCharacters(str string) (string, error) {
 				return 'z'
 			case 'ź':
 				return 'z'
+			case '?':
```
Comment (gbaranski) [inline utils/utils.go]: thats not the best idea i think, we could just remove those characters instead replacing with whitespace.

**Review by gbaranski — CHANGES_REQUESTED** [review]

File: utils/utils.go (hunk 2 above)
Comment (fhodun) [reply utils/utils.go]: ok

File: utils/utils.go [hunk 3]
```diff
@@ -42,13 +44,15 @@ func RemoveUnwantedCharacters(str string) (string, error) {
...
-				return ' '
 			}
 			return r
 		}),
 		norm.NFC,
 	)
 	res, _, err := transform.String(trans, str)
+
+	// Remove question marks from parsed sentence
+	res = strings.Replace(res, "?", "", -1)
```
Comment (gbaranski) [inline utils/utils.go]: instead `strings.Replace` use `strings.ReplaceAll`

**Review by gbaranski — CHANGES_REQUESTED** [review]

**Review by gbaranski — APPROVED** [review]

**Pull request merged**